manifests/0000_90_ingress-operator_03_prometheusrules.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: ingress-operator
+  namespace: openshift-ingress-operator
+  labels:
+    role: alert-rules
+spec:
+  groups:
+    - name: openshift-ingress.rules
+      rules:
+      - alert: HAProxyHighRoute5xxResp
```

Comment:
I would expect every OCP component, that creates a route out of the box, to create their alerting rules like that.
There are some reasons for this:
* that rule does not tell us anything about the health of the router
* components can have alerts defined based on their individual SLOs and have those rules to be more sophisticated because they can be more specific

Reply:
Right, now that I think about it, this would make much more sense.